tests/arch/arm/arm_interrupt/src/arm_interrupt.c

```diff
@@ -377,7 +377,21 @@ void test_arm_interrupt(void)
 	 * entry will make PSP descend below the limit and into the MPU guard
 	 * section (or beyond the address pointed by PSPLIM in ARMv8-M MCUs).
 	 */
+#if defined(CONFIG_FPU) && defined(CONFIG_FPU_SHARING) && \
```

Comment:
This is somewhat redundant because `CONFIG_FPU_SHARING` depends on `CONFIG_FPU`.

Reply:
Riiiight, but it is really everywhere in the code, i.e. the two defines together. Maybe we should do a tree-wide cleanup for this...